### ISSUE TITLE
Does not build since skipper changed to lowercase import

### ISSUE THREAD

**Issue by vetinari** [zalando-incubator/skoap#15]
```
~/.../github.com/zalando-incubator/skoap [skoap:master]$ go get -v ./...
github.com/Sirupsen/logrus (download)
can't load package: package github.com/zalando-incubator/skoap/cmd/skoap: case-insensitive import collision: "github.com/Sirupsen/logrus" and "github.com/sirupsen/logrus"
```

**Comment by vetinari** [comment]
@gargravarr is preparing PR

**Comment by vetinari** [comment]
fixed by #16
